{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/templates/mytemplate.mako

```diff
@@ -1,5 +1,5 @@
 <%inherit file="layout.mako"/>
 <div class="content">
   <h1><span class="font-semi-bold">Pyramid</span> <span class="smaller">Starter project</span></h1>
-  <p class="lead">Welcome to <span class="font-normal">{{cookiecutter.project_name}}</span>, a&nbsp;Pyramid application generated&nbsp;by<br><span class="font-normal">Cookiecutter</span>.</p>
+  <p class="lead">Welcome to <span class="font-normal">{{ "${project}" }}</span>, a&nbsp;Pyramid application generated&nbsp;by<br><span class="font-normal">Cookiecutter</span>.</p>
```

Comment:
I just noticed that you do not need to escape Mako or Chameleon template variables in cookiecutter, but it is necessary for Jinja2 (since it uses Jinja2). I tested locally to confirm. This should be a plain old:

```
            <p class="lead">Welcome to <span class="font-normal">${project}</span>, a&nbsp;Pyramid
```

Ditto for the Chameleon template